pkg/userdata/convert/ignition-converter.go

```diff
@@ -55,7 +55,7 @@ func ToIgnition(s string) (string, error) {
 		return "", fmt.Errorf("failed to convert container linux config to ignition: %s", report.String())
 	}
 
-	out, err := json.Marshal(ignCfg)
+	out, err := json.MarshalIndent(ignCfg, "", "  ")
```

Comment:
Please revert this line here as it applies Indent to format the output and this increases the size of the generated cloud config. AWS cloud provider doesn't permit more than 16384 bytes

Reply:
damn, nice catch!